test/javascripts/acceptance/user-activity-solved-test.js.es6

```diff
@@ -0,0 +1,29 @@
+import { acceptance, exists, query } from "discourse/tests/helpers/qunit-helpers";
+import { test } from "qunit";
+import { visit } from "@ember/test-helpers";
+
+acceptance("Discourse Solved Plugin | activity/solved | empty state", function (needs) {
+  needs.user();
+
+  needs.pretender((server, helper) => {
+    const emptyResponse = { user_actions: [] };
+
+    server.get("/user_actions.json", () => {
+      return helper.response(emptyResponse);
+    });
+  });
+
+  test("When looking at own activity it renders the empty state panel", async function (assert) {
+    await visit("/u/eviltrout/activity/solved");
+    assert.ok(exists("div.empty-state"));
+  });
+
+  test("When looking at another user activity it renders the 'No activity' message", async function (assert) {
+    await visit("/u/charlie/activity/solved");
+    assert.ok(exists("div.alert-info"));
+    assert.equal(
+      query("div.alert-info").innerText.trim(),
+      I18n.t("user_activity.no_activity_others")
+    );
```

Comment:
I know @eviltrout has been removing assertions that depend on translations, though I see the value here in confirming that it displays the correct error message. @eviltrout, what would you say is the best practice?

Reply:
Hmm, this sounds reasonable to me. Some time ago, I started to avoid using such checks in most cases. I feel like it's worth removing it here as well.